ggml/src/ggml-opencl/ggml-opencl.cpp

```diff
@@ -2553,6 +2629,41 @@ static bool ggml_opencl_supports_op(ggml_backend_dev_t dev, const struct ggml_te
             return op->src[0]->type == GGML_TYPE_F32;
         case GGML_OP_SUM_ROWS:
             return op->src[0]->type == GGML_TYPE_F32 && ggml_is_contiguous(op->src[0]);
+        case GGML_OP_FLASH_ATTN_EXT:
+            {
```

Comment:
Sinks support was added last week for gpt-oss-20b. I think we need to mark FA with sinks as unsupported for now; otherwise there will be unit test failures.

```c++
if (op->src[4]) {
    return false;
}
```